src/settings/Settings.tsx

```diff
@@ -436,6 +436,13 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td("Favourite Messages (under active development)"),
         default: false,
     },
+    "feature_new_device_manager": {
+        isFeature: true,
+        labsGroup: LabGroup.Experimental,
```

Comment:
Experimental doesn't sound exactly right... I thought about encryption but I am not sure if that is better

Reply:
Yeah, I had a hard time picking a group for this. Since it's only intended to be labsed during development I felt like this was the best fit.